### PR TITLE
MGDCTRS-1869 Upgrade Debezium Operator to use new storage config options

### PR DESCRIPTION
**Pull request by ani-sha**
https://issues.redhat.com/browse/MGDCTRS-1869

### CODE REVIEW

**Review by Naros — APPROVED** [review]
This LGTM @ani-sha, I re-started the build checks however to see if they pass.

**Comment by rinaldodev** [comment]
There is a bug in the build-it pipeline. I'm sending a fix.

**Review by Naros — APPROVED** [review]
The new `database.encrypt` change LGTM, waiting on CI, and if that passes, will merge.

**Pull request merged**
